src/Session.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace Ohanzee\Helper;
+
+class Session 
+{
+    /**
+      * Returns current session id
+      *
+      *
+      * @return  session_id || boolean(false)
+      *
+      */
+    public static function getId()
+    {
+        return session_id() ?: false;
+    }
+
+    /**
+      * Override current session_id
+      *
+      *
+      * @param string $session_id
+      *
+      * @return  session_id || boolean(false)
+      *
+      */
+    public static function setId($session_id)
+    {
+        return self::isActive() ? false : session_id($session_id);
```

Comment:
Now that I'm looking at it, i'm not really sure what happens if you try to set a session id while the session is already started. Doesn't mention anything in the docs. I'm guessing you would lose current session data, but i'll have to look into this further. I'm going to add the LogicException for now.